src/timestamp.ts

```diff
@@ -0,0 +1,5 @@
+let timestamp = 1;
```

Comment:
Start from `-1`?

Maybe this file should be called `clock.ts` or something similar? This module itself isn't really a timestamp.